src/multiqc.sh

```
@@ -36,11 +36,16 @@ main() {
 
     cat input_files.txt | xargs -P$(nproc --all) -n1 -I{} dx download {} -o ./inputs/
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)